go.mod

```diff
@@ -9,9 +9,9 @@ require (
 	github.com/casbin/casbin/v2 v2.35.0
 	github.com/coreos/go-semver v0.3.0
 	github.com/ethereum/go-ethereum v1.13.4
-	github.com/ethersphere/go-price-oracle-abi v0.1.0
-	github.com/ethersphere/go-storage-incentives-abi v0.6.0
-	github.com/ethersphere/go-sw3-abi v0.4.0
+	github.com/ethersphere/go-price-oracle-abi v0.2.0
+	github.com/ethersphere/go-storage-incentives-abi v0.6.1-rc3
```

Comment:
here we are using an -rc tag, is this safe to be merged to master and released?

Reply:
RC is because the actual release is still on goerli as when you built nodes so far you used it and If sepolia was in release then you would have problem.  If we are ready I can add it to official release also and different TAG can be used.

Reply:
If we are going to merge this to master then we probably shouldn't be using RCs

Reply:
Not a problem, here is non-RC release

https://github.com/ethersphere/go-storage-incentives-abi/tree/v0.6.2